src/core_plugins/kbn_vislib_vis_types/public/gauge.js

```diff
@@ -23,15 +23,16 @@ import { CATEGORY } from 'ui/vis/vis_category';
 import gaugeTemplate from './editors/gauge.html';
 import { vislibColorMaps } from 'ui/vislib/components/color/colormaps';
 
-export default function GaugeVisType(Private) {
+export default function GaugeVisType(Private, i18n) {
   const VisFactory = Private(VisFactoryProvider);
 
   return VisFactory.createVislibVisualization({
     name: 'gauge',
-    title: 'Gauge',
+    title: i18n('kbnVislibVisTypes.gauge.gaugeTitle', { defaultMessage: 'Gauge' }),
     icon: 'visGauge',
-    description: `Gauges indicate the status of a metric. Use it to show how a metric's value relates
-      to reference threshold values.`,
+    description: i18n('kbnVislibVisTypes.gauge.gaugeDescription', {
+      defaultMessage: 'Gauges indicate the status of a metric. Use it to show how a metric\'s value relatesto reference threshold values.'
```

Comment:
To prevent the need of escaping the single quote, please just use a backtick string here.

Reply:
@timroes there will be error.

Reply:
Just a heads up that there's a missing space here between `relatesto`

Reply:
@lukeelmers fix it, thanks